tests/cephfs/quota/quota_files.py

```diff
@@ -0,0 +1,170 @@
+import logging
+import random
+import string
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Test Cases Covered :
+    CEPH-83573399	Test to validate the quota.max_files
+                    Create a FS and create 10 directories and mount them on kernel client and fuse client(5 mounts each)
+                    Set max files quota to a number(say 50) and add up to that number of files to that directory and
+                    verify if the set quota limit is working fine. Similarly set different limit
+                    on different directories and verify quota.
+
+    Pre-requisites :
+    1. We need atleast one client node to execute this test case
+    2. create fs volume create cephfs if the volume is not there
+    3. ceph fs subvolumegroup create <vol_name> <group_name> --pool_layout <data_pool_name>
+        Ex : ceph fs subvolumegroup create cephfs subvolgroup_clone_attr_vol_1
+    4. ceph fs subvolume create <vol_name> <subvol_name> [--size <size_in_bytes>] [--group_name <subvol_group_name>]
+       [--pool_layout <data_pool_name>] [--uid <uid>] [--gid <gid>] [--mode <octal_mode>]  [--namespace-isolated]
+       Ex: ceph fs subvolume create cephfs subvol_1 --size 5368706371 --group_name subvolgroup_1
+    5. ceph fs subvolume create <vol_name> <subvol_name> [--size <size_in_bytes>] [--group_name <subvol_group_name>]
+       [--pool_layout <data_pool_name>] [--uid <uid>] [--gid <gid>] [--mode <octal_mode>]  [--namespace-isolated]
+       Ex: ceph fs subvolume create cephfs subvol_2 --size 5368706371 --group_name subvolgroup_1
+
+    Test Case Flow:
+    1. Mount the subvolume_1 on the client using fuse
```

Comment:
Quota is applicable on regular Directories under the root filesystem also.

Can you also add few directories directly on root and use them along with the directories under the subvolumes. ?

Reply:
I have added Directories to the root volume and validated the file quota